settings.ts

```diff
@@ -71,6 +71,8 @@ export const PUBLIC_TAG_PARENT_IDS: number[] = [
     1510
 ]
 
+export const EPOCH_DATE = "2020-01-21"
```

Comment:
I would be inclined to go with another epoch date.
I realize that's purely a taste question, but I just find `2020-01-21` looks odd and will forever be a historical weirdness in the codebase.
I would prefer some "more normal-looking" date, like `2020-01-01`, `2000-01-01`, or even `1970-01-01`.
We already do a mapping, so even the existing charts shouldn't be a huge problem. And we could also run a one-time database migration to normalize all database entries to the same epoch date, which also eliminates the need for `display.epochDate` at all (which might be the goal anyway?).

Reply:
Yes I agree it's awkward from a development point of view, and the eventual goal is to get rid of this historical weirdness by changing how we deal with the time dimension in the Grapher. This day-offset approach allowed us to quickly implement dates but isn't a good long-term solution, since we need to deal with more temporal granularities, e.g. monthly.

One reason for going with `2020-01-21` is to keep past URLs working. But as you say, we currently map these so we can just add another "old epoch" constant and shift variables relative to that constant for URL params.

But another bigger reason for going with `2020-01-21` is that it has become our de-facto epoch for COVID data. All the different scripts we have for updating the daily data are using that date, specifically to avoid the problem that this PR is fixing – that you can't combine variables with different `zeroDay`s.

I think for the time being it's better to stick with this date and not introduce any changes for the authors who upload data daily. 

This PR takes a step in making it easier to change the date-handling logic in the future by not exposing the day offsets in the URL. Another step we should soon make is to handle ISO-formatted dates at the import step, so that authors stop needing to transform dates to offsets, and they can stop dealing with offsets too.

After we hide this implementation detail from both users and authors, we will be able to change our implementation more easily. Until then... we will be duct-taping. 😄